simapp/app.go

```diff
@@ -503,9 +503,11 @@ func (app *SimApp) SimulationManager() *module.SimulationManager {
 // RegisterAPIRoutes registers all application module routes with the provided
 // API server.
 func (app *SimApp) RegisterAPIRoutes(apiSvr *api.Server) {
-	rpc.RegisterRoutes(apiSvr.ClientCtx, apiSvr.Router)
-	authrest.RegisterTxRoutes(apiSvr.ClientCtx, apiSvr.Router)
-	ModuleBasics.RegisterRESTRoutes(apiSvr.ClientCtx, apiSvr.Router)
+	clientCtx := apiSvr.ClientCtx
+	clientCtx = clientCtx.WithJSONMarshaler(clientCtx.LegacyAmino)
```

Comment:
ditto

Reply:
Yep for REST routes we need amino. I'll add comments

Reply:
Comment added 